import-automation/cloudbuild/create_task.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Creates a Cloud Tasks task that ships information about a GitHub commit
+to the executor.
+"""
+
+import os
+import json
+
+from google.cloud import tasks_v2
+
+
+TASK_BODY_FIELDS = [
+    'COMMIT_SHA',
+    'REPO_NAME',
+    'BRANCH_NAME',
+    'HEAD_BRANCH',
+    'BASE_BRANCH',
+    'PR_NUMBER'
+]
+
+
+def create_body():
+    """Creates the task body for a Cloud Tasks task.
+
+    Specifically, this function looks for the value of every field defined
+    in TASK_BODY_FIELDS in the environment variables.
+
+    PR_NUMBER is converted to an integer.
+
+    Returns:
+        The task body as a dict.
+    """
+    task_body = {}
+    for field in TASK_BODY_FIELDS:
+        task_body[field] = os.environ[field]
```

Comment:
What happens when a particular field is not in os.environ?  Should get() be used?

Reply:
These fields are required. If a field is missing, the script will fail.

Reply:
Let's write a comment to that effect, making it explicit.

Reply:
added